dspace/src/main/docker/README.md

```diff
@@ -130,6 +130,23 @@ docker run -i -t -d -p 80:80 -p 443:443 dspace/dspace-shibboleth
 
 This image can also be rebuilt using the `../docker-compose/docker-compose-shibboleth.yml` script.
 
+## dspace/src/main/docker/dspace-solr/Dockerfile
+
+This Dockerfile builds a Solr image with DSpace Solr configsets included. It
+can be pulled / built following the [docker compose resources](../docker-compose/README.md)
+documentation. Or, to just build and/or run Solr:
+
+```bash
+docker compose build dspacesolr
+docker compose up -d dspacesolr
+```
+
+If you're making iterative changes to the DSpace Solr configsets you'll need to rebuild /
+restart the `dspacesolr` container for the changes to be deployed. From DSpace root:
+
+```bash
+docker compose up --detach --build dspacesolr
```

Comment:
Similar to above, I think this should be aligned with other docker compose instructions.  This works for me:
```
docker-compose -p d7 up -d --build dspacesolr
```

Reply:
Done!